docs/example_navigation_ros_codegen.md

```diff
@@ -0,0 +1,527 @@
+---
+id: example_navigation_ros_codegen
+title: Navigation of ground vehicle using ROS codegen
+description: Nonlinear model predictive control with OpEn using ROS
+---
+
+<script type="text/x-mathjax-config">MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});</script>
+<script type="text/javascript" async src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
+
+Example contributed by Guido Sanchez ([Github](https://github.com/gmsanchez/))
+
+## Auto-generated ground vehicle navigation for ROS
+
+Consider the navigation problem for an autonomous ground vehicle that can be modeled using the differential drive equations.
```

Comment:
Can you add the equations of motion? You can just add the following (check whether it is correct):

```html
<div class="math">
\[\begin{split}
\dot{x}(t) {}={}&amp; u(t) \cos \theta(t),\\
\dot{y}(t) {}={}&amp; u(t) \sin \theta(t),\\
\dot{\theta}(t) {}={}&amp; \alpha(t).
\end{split}\]</div>
```